tests/geo_image_test.py

```diff
@@ -411,3 +411,33 @@ def test_clipping_rotated_image(tmp_path):
             grid.u_count,
             4,
         )
+
+
+def test_image_rotation(tmp_path):
+    workspace = Workspace(tmp_path / r"geo_image_test.geoh5")
+
+    # Repeat with gray scale image
+    image = Image.fromarray(np.random.randint(0, 255, (128, 128)).astype("uint8"), "L")
+    geoimage = GeoImage.create(workspace, name="test_area", image=image)
+
+    np.testing.assert_array_almost_equal(geoimage.rotation, 0)
+    np.testing.assert_array_almost_equal(geoimage.dip, 0)
+
+    geoimage2 = geoimage.copy()
+    geoimage2.rotation = 66
+
+    np.testing.assert_array_almost_equal(geoimage2.rotation, 66)
+
+    geoimage3 = geoimage.copy()
+
+    geoimage3.dip = 44
+
+    np.testing.assert_array_almost_equal(geoimage3.dip, 44)
+
+    geoimage4 = geoimage.copy()
+
+    geoimage4.dip = 44
+    geoimage4.rotation = 66
+
+    np.testing.assert_array_almost_equal(geoimage4.dip, 44)
+    np.testing.assert_array_almost_equal(geoimage4.rotation, 66)
```

Comment:
Still only checking the values have changed, but we should also test the side effects on `vertices`, and vice versa.